keel-core/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DockerArtifact.kt

```diff
@@ -16,14 +17,19 @@ data class DockerArtifact(
   override val name: String,
   override val deliveryConfigName: String? = null,
   override val reference: String = name,
-  val tagVersionStrategy: TagVersionStrategy = SEMVER_TAG,
+  val tagVersionStrategy: TagVersionStrategy? = null,
   val captureGroupRegex: String? = null,
-  override val from: ArtifactOriginFilterSpec? = null
+  val branch: String? = null,
```

Comment:
Convenience property to indirectly set the `ArtifactOriginFilter` below without typing a bunch of stuff.